### DISCUSSION TITLE
apt not upgrading onedrive

### DISCUSSION THREAD

**Discussion by deron-dev** [abraunegg/onedrive#1669] [Q&A] [answered]
When I run `sudo apt upgrade` on Ubuntu 21.10, my packages upgrade other than onedrive. I get this message:
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Calculating upgrade... Done
The following packages have been kept back:
  onedrive
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
```
Is this desired, or should I force the upgrade? Any guidance would be appreciated :)

**Comment by abraunegg** [comment, marked as answer]
@terminal-root 
What is your current OneDrive application | package version  - `onedrive --version` ?

**Reply by deron-dev** [reply]
`onedrive v2.4.13-1~np1`

**Reply by abraunegg** [reply]
That is the latest version of the client for Ubuntu platforms and aligns to the latest release as per https://github.com/abraunegg/onedrive/releases

**Reply by deron-dev** [reply]
Hm, okay, I will keep an eye on releases and see if the behavior changes on a new release. Thank you much :)
